UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/appsettings.json

```diff
@@ -65,8 +65,9 @@
     "LargeExchangeSetAccountKey": "",
     "LargeExchangeSetInstance": 1,
     "StorageContainerName": "ess-fulfilment",
-    "QueueName": "ess-1-fulfilment",
-    "LargeExchangeSetSizeInMB": 300
+    "DynamicQueueName": "ess-{0}-fulfilment",
```

Comment:
As we have done changes for dynamic queue and storage based on exchange set size, can we remove below from appsettings.json
"StorageAccountName": "",
    "StorageAccountKey": "", 
Also make necessary changes in appsettings.local.overrrides.json to keep both settings in sync

Reply:
Yes removed unused properties